cfme/tests/infrastructure/test_providers.py

```diff
@@ -375,6 +376,48 @@ def _refresh_provider():
     assert gd_count_before == gd_count_after, "guest devices count changed after refresh!"
 
 
+@pytest.mark.rhv2
+@test_requirements.rhev
+@pytest.mark.meta(automates=[1594817])
+@pytest.mark.provider([RHEVMProvider], selector=ONE, scope="function")
+def test_rhv_custom_attributes_after_refresh(appliance, setup_provider, provider):
+    """
+    Bugzilla:
+        1594817
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Infra
+        caseimportance: high
+        initialEstimate: 1/8h
+        testSteps:
+            1. Create a custom attribute on a vm
+            2. Run a targeted refresh of the VM
+            3. Check if the custom attribute is still there
+        expectedResults:
+            1.
+            2.
+            3. The custom attribute is still there
+    """
+    # get the name of any VM on the provider
+    view = navigate_to(provider, 'ProviderVms')
+    vm_name = view.entities.all_entity_names[0]
+
+    vm = f"Vm.where(name: '{vm_name}').last"
+
```

Comment:
**Optional:**
It would be better if we remove **vm_name** variable. As it used once. So we can simply go with: ```vm = f"Vm.where(name: '{view.entities.all_entity_names[0]}').last"```